mercure.rst

```diff
@@ -333,9 +362,9 @@ by using the ``AbstractController::addLink`` helper method::
 
     class DiscoverController extends AbstractController
     {
-        public function __invoke(Request $request, Discovery $discovery): JsonResponse
+        public function discover(Request $request, Discovery $discovery): JsonResponse
         {
-            // Link: <http://localhost:3000/.well-known/mercure>; rel="mercure"
+            // Link: <http://hub.example.com/.well-known/mercure>; rel="mercure"
```

Comment:
Which of `http` and `https` is the right scheme using the flex integration? There's a mix both in the document, can we stick to one and maybe mention that the hub should be served over TLS in prod?

Reply:
By the spec, HTTPS is mandatory (in prod at least). The Docker integration doesn't support HTTPS out of the box yet, but I hope that we'll be able to fix that at some point. I updated all the URLs to use HTTPS, as it should be in prod.